readme.rst

```diff
@@ -31,18 +31,17 @@ New Botan releases will be audited on a regular basis and cryptographically rele
 changes will be checked and documented. These releases will eventually be
 released here when approved by the BSI.
 
-New releases are also announced on the botan-bsi-announce@rohde-schwarz.com mailing list.
-Subscribe by sending an email to botan-bsi-announce-register@rohde-schwarz.com.
-
 Release Downloads
 ----------------------------------------
 
 The latest version is `2.4.0-RSCS1 <https://cdn.rohde-schwarz.com/pws/dl_downloads/dl_software/zip/Cybersecurity-software-Botan-2.4.0-RSCS1.zip>`_ (`sig <https://cdn.rohde-schwarz.com/pws/dl_downloads/dl_software/sig/Cybersecurity-software-Botan-2.4.0-RSCS1.zip.sig>`_) released on 2018-09-28.
 
 `SHA-256 <https://cdn.rohde-schwarz.com/pws/dl_downloads/dl_software/sha256/Cybersecurity-software-Botan-2.4.0-RSCS1.zip.sha256>`_: EADC9A20FBE6AF4725CBA26B0636C1244083BAF4F936A129AF498D056EACBF75
 
-All approved releases are signed with the following `key <https://cybersecurity.rohde-schwarz.com/sites/default/files/sha_downloads/Botan-Signing-Key.pem>`_ (`cert <https://cybersecurity.rohde-schwarz.com/sites/default/files/sha_downloads/Botan-Signing-Cert.pem>`_)::
+All approved releases are signed with the following key::
 
+  Botan-Signing-Key.pem
```

Comment:
What's that file name for? Did you want to link it to something maybe?

Reply:
Previously we linked to the PEM file on our website, too. That was its filename. Since the link was broken I removed it, but the filename is referenced in the verification code below, so I did want to keep it.

Reply:
I now changed the section below to refer to the filename and removed it from the block.